### PR TITLE
(web components): add readme.md files for each component

### LINKED ISSUE #17318: Add readme for all fluent web components
#### Describe the feature that you would like added

Add readme for all web components

### PR DESCRIPTION
**Pull request by brookdozer**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #17318
- [x] Include a change request file using `$ yarn change`

Added README.md files for each Web Component.

### CODE REVIEW

**Comment by chrisdholt** [comment]
> How should we handle the component explorer resource links?

@brookdozer we should definitely drop the links to the component explorer. I assume any examples like that would be handled on the Fluent site and any additional examples we are looking at providing via embedded "usage" examples potentially.

**Comment by brookdozer** [comment]
> > How should we handle the component explorer resource links?
> 
> @brookdozer we should definitely drop the links to the component explorer. I assume any examples like that would be handled on the Fluent site and any additional examples we are looking at providing via embedded "usage" examples potentially.

I'll refine the links/resources sections at the bottom of each README. We _should_ have embedded usage examples in the Fluent site but they may not be ready before v1 launch, so additional supporting examples like the FAST documentation or the  Fluent Storybook wouldn't hurt at this stage, so long as we make it clear that they're supporting examples and that the main live examples will be in the Fluent site pages

File: packages/web-components/src/accordion/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+As defined by the [W3C](https://w3c.github.io/aria-practices/#accordion):
+
+> An accordion is a vertically stacked set of interactive headings that each contain a title, content snippet, or thumbnail representing a section of content. The headings function as controls that enable users to reveal or hide their associated sections of content. Accordions are commonly used to reduce the need to scroll when presenting multiple sections of content on a single page.
+
+## Setup
+
+### Basic Setup
+
+```ts
+import { providefluentDesignSystem, fluentAccordion, fluentAccordionItem } from '@fluentui/web-components';
```
Comment (SethDonohue) [inline packages/web-components/src/accordion/README.md:18]:
```suggestion
import { provideFluentDesignSystem, fluentAccordion, fluentAccordionItem } from '@fluentui/web-components';
```

Comment (SethDonohue) [reply packages/web-components/src/accordion/README.md:18]: same in other places where `providefluentDesignSystem` is.

**Review by SethDonohue — APPROVED** [review]

**Comment by Pearseak** [comment]
Is the web component package has all react components?

**Comment by chrisdholt** [comment]
> Is the web component package has all react components?

Not sure I fully understand the question as there's several versions of Fluent UI React. I can say we are aligned on a baseline set of controls with the upcoming vNext react work. If you have other questions, please feel free to file an issue!

File: packages/web-components/src/accordion/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  template,
+  styles,
+  collapsedIcon: `...default collapsed icon...`,
+  expandedIcon: `...default expanded icon...`,
+});
+```
+
+## Additional resources
+
+- [Component explorer examples](https://explore.fast.design/components/fast-accordion)
```
Comment (chrisdholt) [inline packages/web-components/src/accordion/README.md:88]: @brookdozer we're still planning on removing these links, yes?

**Pull request closed without merging**